portal/src/components/Nav.vue

```diff
@@ -187,15 +222,37 @@ nav {
 	padding: 0.5em;
 	border-radius: 0.5em;
 	place-self: center;
+
 	&:hover {
 		border-color: #303030;
 	}
+
 	&:active {
 		background-color: #424242;
 	}
 }
 
 @media screen and (max-width: 1000px) {
```

Comment:
Why have this breakpoint if we are going to use breakpoints under it?

Reply:
Not sure, the @media tag was there before I started working on this, I will optimize it.